pkg/gofr/gofr_test.go

```diff
@@ -746,3 +914,31 @@ func Test_Shutdown(t *testing.T) {
 
 	assert.Contains(t, logs, "Application shutdown complete", "Test_Shutdown Failed!")
 }
+
+func TestApp_Subscribe(t *testing.T) {
```

Comment:
We should rename the test method to `TestApp_SubscriberInitialize` since it is not using the mock subscriber or actually subscribing but just checking the other thing if the subscriber is initialized successfully.